commands/audit/jas/secrets/secretsscanner.go

```diff
@@ -12,14 +12,16 @@ import (
 )
 
 const (
-	secretsScanCommand   = "sec"
-	secretsScannerType   = "secrets-scan"
-	secretsDocsUrlSuffix = "secrets"
+	secretsScanCommand           = "sec"
+	SecretsScannerType           = "secrets-scan"        // #nosec
+	SecretsScannerDockerScanType = "secrets-docker-scan" // #nosec
```

Comment:
```suggestion
	SourceCodeScan SecretsScanType = "secrets-scan"        // #nosec
	DockerScan     SecretsScanType = "secrets-docker-scan" // #nosec
```
Consider defining a type: `type SecretsScanType string`

Reply:
Done